templates/config/prometheus.yml.tpl

```diff
@@ -376,3 +376,35 @@ scrape_configs:
       - '{{.}}'
     {{- end}}
 {{- end}}
+
+{{- if .RemoteWriteURL}}
+remote_write:
+   - url: {{.RemoteWriteURL}}
+     queue_config:
```

Comment:
IMHO, the full config of `remote_write` such as `remote_timeout`, `write_relabel_configs` should also be imported.

Reply:
fixed